master/getting-started/bare-metal/bare-metal.md

```diff
@@ -181,57 +202,61 @@ Once you have such a policy in place, you may want to disable the
 >
-> :   The selector in the policy, `all()`, will match *all* endpoints, 
->     including any workload endpoints. If you have workload endpoints as 
->     well as host endpoints then you may wish to use a more restrictive 
->     selector. For example, you could label management interfaces with 
->     label `endpoint_type = management` and then use selector 
->     `endpoint_type == "management"`
->
-> :   If you are using Calico for networking workloads, you should add
->     inbound and outbound rules to allow BGP, for example:
+> The selector in the policy, `all()`, will match *all* endpoints, 
+> including any workload endpoints. If you have workload endpoints as 
+> well as host endpoints then you may wish to use a more restrictive 
+> selector. For example, you could label management interfaces with 
+> label `endpoint_type = management` and then use selector 
+> `endpoint_type == "management"`
 >
-> :   `{"protocol": "tcp", "dst\_ports": \[179\], "action": "allow"}`
+> If you are using Calico for networking workloads, you should add
+> inbound and outbound rules to allow BGP:  add an ingress and egress rule
+> to allow TCP traffic to destination port 179.
 
 ## Creating host endpoint objects
 
 For each host endpoint that you want Calico to secure, you'll need to
-create a host endpoint object in etcd. At present, this must be done
-manually using `etcdctl set <key> <value>`.
+create a host endpoint object in etcd.  Use the `calicoctl create` command
+to create a host endpoint resource (hostEndpoint).
 
 There are two ways to specify the interface that a host endpoint should
 refer to. You can either specify the name of the interface or its
-expected IP address. In either case, you'll also need to know the
-hostname of the host that owns the interface.
+expected IP address. In either case, you'll also need to know the name given to
+the Calico node running on the host that owns the interface; in most cases this
+will be the same as the hostname of the host.
 
 For example, to secure the interface named `eth0` with IP 10.0.0.1 on
-host `my-host`, you could create a host endpoint object at
-`/calico/v1/host/<hostname>/endpoint/<endpoint-id>` (where `<hostname>`
-is the hostname of the host with the endpoint and `<endpoint-id>` is an
-arbitrary name for the interface, such as "data-1" or "management") with
-the following data:
-
-    {
-      "name": "eth0",
-      "expected_ipv4_addrs": ["10.0.0.1"],
-      "profile_ids": [<list of profile IDs>],
-      "labels": {
-        "role": "webserver",
-        "environment": "production",
-      }
-    }
+host `my-host`, run the command below.  The name of the endpoint is an 
+arbitrary name required for endpoint identification.
+
+When running this command, replace the placeholders in angle brackets with
+appropriate values for your deployment.
+
+```
+cat << EOF | calicoctl create -f -
+- apiVersion: v1
+  kind: hostEndpoint
+  metadata:
+    name: <name of endpoint>
```

Comment:
not necessarily.  YAML is funny like that.  Adding quotes around a string is more reliable, but I'm not convinced I want to get into a what is valid YAML discussion because it's a complicated topic :-)